src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -304,15 +327,20 @@ public bool IsAtLeastAsVisibleAs(Symbol sym, ref HashSet<DiagnosticInfo> useSite
             return NullableUnderlyingTypeOrSelf.IsAtLeastAsVisibleAs(sym, ref useSiteDiagnostics);
         }
 
-        public virtual TypeSymbolWithAnnotations SubstituteType(AbstractTypeMap typeMap)
```

Comment:
>virtual [](start = 15, length = 7)

What's the virtual removal change related to?

Reply:
The new `protected` overload is now virtual so derived classes have a single method to override rather than two.